src/pilet-build.test.ts

```diff
@@ -271,13 +271,19 @@ runTests('pilet-build', ({ test, setup }) => {
     'from-sample-piral-without-minify',
     'can build a pilet without minify',
     ['pilet.v2', 'build.pilet'],
-    async (ctx) => {
-      await ctx.run(`npx pilet build --no-minify`);
+    async (ctx) => {      
+      await ctx.setFiles({
+        'src/index.tsx': `
+          function foobar1234() { return 42; }
```

Comment:
Ouch - danger! You are replacing the content here!

It's fine right now, but in the future we could actually error out here because no `setup` is exported.